app/api/__init__.py

```diff
@@ -47,7 +47,7 @@
 # tickets
 api.route(AllTicketList, 'all_ticket_list', '/tickets', '/events/<int:id>/tickets', '/events/<identifier>/tickets')
 api.route(TicketDetail, 'ticket_detail', '/tickets/<int:id>', '/events/<int:event_id>/tickets',
-          '/attendees/<int:attendee_id>/ticket')
+          '/events/<event_identifier>/tickets', '/attendees/<int:attendee_id>/ticket')
```

Comment:
This is ticket detail. Why is `/events/<event_identifier>/tickets` needed here when it's already there for TicketList ?